src/App.js

```diff
@@ -3,18 +3,20 @@ import { ChakraProvider } from "@chakra-ui/react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { Home } from "./pages/home";
 import { QueryClient, QueryClientProvider } from "react-query";
-
+import Layout from "./components/layout";
 const queryClient = new QueryClient();
 
 function App() {
   return (
     <ChakraProvider>
       <QueryClientProvider client={queryClient}>
         <BrowserRouter>
-          <Routes>
-            <Route path="/" element={<Home />} />
-            <Route path="about" element={<h1>About</h1>} />
-          </Routes>
+          <Layout>
+            <Routes>
+              <Route path="/" element={<Home />} />
+              <Route path="about" element={<h1>About</h1>} />
+            </Routes>
+          </Layout>
```

Comment:
Layout should be used for each page so instead of making it a parent of `<Routes>`, we use it in each page component:

```js
export function Home() {
  return (
    <Layout>
    ...
    </Layout>
  );
}
```